### PR TITLE
Add Storybook documentation re the new form defaults file

### LINKED ISSUE #1827: Add Storybook documentation re the new form defaults file
## Summary

document the new process for changing fields introduced in #1810

### PR DESCRIPTION
**Pull request by kevinyang06**
Fixes #1827

# Description

> document the new process for changing fields introduced in #1810

# Any new packages installed?

> no

# Required followup work

> no

# Checklist:

- [ ] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [ ] I have added and needed tests for my changes (in particular for new screens)
- [ ] I have added a comment to any confusing code
- [ ] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

File: f2/src/__stories__/Forms.stories.mdx [hunk 1]
```diff
@@ -143,6 +143,13 @@ View [Field](/?path=/docs/components-field--input-field) documentation for more
 
 View [FormArrayControl](/?path=/docs/components-form-array-control--default-story) documentation for more information on how to place radio and checkbox lists in a Form.
 
+## Default Values
+
+formDefaults is gaint Javascript object contains a list of all fields from report with empty values asigned for each field.
```
Comment (amazingphilippe) [inline f2/src/__stories__/Forms.stories.mdx:148]: Is "gaint" a typo here?

File: f2/src/__stories__/Forms.stories.mdx [hunk 2]
```diff
@@ -143,6 +143,13 @@ View [Field](/?path=/docs/components-field--input-field) documentation for more
...
+The value can be a empty string or empty array based on the type of field.
```
Comment (amazingphilippe) [inline f2/src/__stories__/Forms.stories.mdx:149]: Can you provide examples of types of field, and how they are implemented in formDefaults? You could put in a small js code block as an example.

File: f2/src/__stories__/Forms.stories.mdx (hunk 1 above)
Comment (kevinyang06) [reply f2/src/__stories__/Forms.stories.mdx]: fixed

File: f2/src/__stories__/Forms.stories.mdx (hunk 2 above)
Comment (kevinyang06) [reply f2/src/__stories__/Forms.stories.mdx:149]: fixed

**Pull request merged**